### DISCUSSION TITLE
一起討論精簡OS

### DISCUSSION THREAD

**Discussion by andypig04** [WhatTheBlock/WindowsSimplify#50] [General]
你好
我也有在自製精簡版 win11 22H2 以自用為主
目前已經同步到官方最新版 22621.382
想一起討論整合 recitfy 11 主題
和一些精簡的相關
是否有其他聯繫方式呢
不吝指教 
謝謝!!

**Comment by WhatTheBlock** [comment]
discord：wtb_

其他看到的人想加我好友也可以
這個issue我就暫時不關閉了

**Comment by andypig04** [comment]
> 小二#5516

社畜上班中...

晚點下班到家加

謝謝~
